backend/src/exceptions/sc-error/sc-error.ts

```diff
@@ -0,0 +1,17 @@
+import { ExceptionMessage, HttpCode } from '~/common/enums/enums';
+
+const DEFAULT_MESSAGE = ExceptionMessage.MASTER_INSTANCE_CREATE;
+
+class SCError extends Error {
+  status: HttpCode;
+
+  constructor({
+    status = HttpCode.BAD_REQUEST,
+    message = DEFAULT_MESSAGE,
+  } = {}) {
```

Comment:
```suggestion
  constructor({
    status = HttpCode.BAD_REQUEST,
    message = DEFAULT_MESSAGE,
  }: Constructor = {}) {
```
And add type Constructor